requirements.txt

```diff
@@ -1,2 +1,4 @@
 pytest==4.1.0
 PyYAML==4.2b4
+# attrs 19.2 not compatable with pytest 4.1
+attrs < 19.2
```

Comment:
Sure, except pytest 4.1 has its dependency at `"attrs>=17.4.0"`, so it will happily pull in the latest version, which happened to introduce a breaking change.